### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wjimenez5271**
highlight community events

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,13 @@ Longhorn implements distributed block storage using containers and microservices
 
 You can read more technical details of Longhorn [here](https://longhorn.io/).
 
+
+## Get Involved
+**Community Office Hours**!: Hosted by the core maintainers of Longhorn: 2nd Friday of the every month at 09:00 Pacific Time (PT)/12:00 Eastern Time (ET) on Zoom: http://bit.ly/longhorn-community-meeting
```
Comment (yasker) [inline README.md:28]: Can we add the calendar (http://bit.ly/longhorn-events) to it as well? Since it's not easy to remember to join once a month.

**Review by yasker — CHANGES_REQUESTED** [review]
Can you also add your `Signed-off-by` ? See https://github.com/longhorn/longhorn/blob/master/CONTRIBUTING.md#dco-sign-off

File: README.md (hunk 1 above)
Comment (yasker) [reply README.md:28]: Also, it's called `Community meeting and office hour`.

Comment (wjimenez5271) [reply README.md]: @yasker thanks updated

**Review by yasker — APPROVED** [review]

**Pull request merged**